tests/fixup/network_configuration.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "x11test";
+use strict;
+use testapi;
+
+sub run() {
+    my $self = shift;
+
+    # openSUSE 13.2's (and earlier) systemd has broken rules for virtio-net, not applying predictable names (despite being configured)
+    # A maintenance update breaking networking names sounds worse than just accepting that 13.2 -> TW breaks with virtio-net
+    # At this point, the system has been updated, but our network interface changed name (thus we lost network connection)
+    my $command = "cp /etc/sysconfig/network/ifcfg-eth0 /etc/sysconfig/network/ifcfg-ens4; /usr/sbin/ifup ens4";
+
+    if (get_var("DESKTOP") =~ /kde|gnome/) {
```

Comment:
I'd prefer not switching up / down to X when not needed - so far, we do not do any update tests of other systems than gnome/kde